src/test/java/com/ibm/cloud/sdk/core/test/service/ErrorResponseTest.java

```diff
@@ -102,18 +152,18 @@ public void testBadRequest() {
   public void testUnauthorized() {
 
     String message = "The request failed because the moon is full.";
+    String responseBody = "{\"message\": \"" + message + "\", \"other_stuff\": \"foo\"}";
```

Comment:
Also, used "message" instead of "error" for the JSON field name since we were not exercising that use-case.